### PR TITLE
Updated & added missing german translations [de-DE]

### PR DESCRIPTION
**Pull request by christianrr**

### CODE REVIEW

File: src/lang/locales/de-DE.js [hunk 1]
```diff
@@ -18,123 +20,215 @@ export const dateTimeFormats = {
...
+      YEAR: "Jahr. Erlaubt ist `1901` bis `2155` or `0000`"
+    }
+  },
+  date_and_time: "Datum und Uhrzeit",
+  db_user: "Benutzer (Datenbank)",
+  db_password: "Benutzerpasswort (Datenbank)",
   db_name: "Name in der Datenbank",
-  db_type: "Datenbank-Typ",
+  db_type: "Datenbanktyp",
+  db_column_name: "Datenbankfeldname"",
```
Comment (shealavington) [inline src/lang/locales/de-DE.js]: Double quotes causing build fail. :)

Comment (rijkvanzanten) [inline src/lang/locales/de-DE.js]:
```suggestion
  db_column_name: "Datenbankfeldname",
```

**Comment by rijkvanzanten** [comment]
Thanks @christianrr! 😄

**Comment by rijkvanzanten** [comment]
Wait where did those conflicts come from?

**Comment by christianrr** [comment]
hmm, weird... the conflicts are the entries I've added!?

**Comment by rijkvanzanten** [comment]
Very strange. Git at it's best I suppose 🤷‍♂ 

Anyway, fixed now. Thanks @christianrr :)

**Comment by rijkvanzanten** [comment]
Ay spoke too soon. There are a couple instances where there are duplicate keys:

image

I don't know which of the two is correct... @christianrr do you mind verifying these?

It's `almost_done_options`, `extension_error`, `related_collection`, `show_directus_collections`, and `this_collection` that are duplicated.

File: src/lang/locales/de-DE.js [hunk 2]
```diff
@@ -124,17 +215,20 @@ export default {
     102: "Aufgrund von Inaktivität abgemeldet",
     103: "Benutzer inaktiv",
     106: "Falscher Benutzername und/oder Passwort"
```
Comment (rijkvanzanten) [inline src/lang/locales/de-DE.js]: Oh and there's a `,` missing after this line 🙂 

```suggestion
    106: "Falscher Benutzername und/oder Passwort",
```

**Comment by christianrr** [comment]
Sorry for the errors; I made my changes literally in a text editor matching the english and the german version for missing translations ;) is there any good tool available to transfer the newly introduced values to the translation files?

**Comment by rijkvanzanten** [comment]
We've been looking at integrating [Crowdin](https://crowdin.com/), but that takes some time to integrate (they use a slightly different structure for translations).

**Pull request merged**